include/spm/mario_status.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <common.h>
+
+CPP_WRAPPER(spm::mario_status)
+
+typedef enum MarioStatusType {
```

Comment:
Since these are bitflags it'd be better to have them in hex rather than decimal, we also usually use #defines rather than enums for flags

Reply:
In ghidra it was set as an enum which is why I thought I'd do it this way, should I just do a #define for each individual variable and change the function from `void marioStatusApplyStatuses(MarioStatusType status, s32 lv);` to `void marioStatusApplyStatuses(s32 status, s32 lv);`?

Reply:
Yeah, ghidra just has an enum because that's the only real way to document it there